plugins/doc_fragments/azure.py

```diff
@@ -69,10 +69,12 @@ class ModuleDocFragment(object):
         description:
             - Controls the source of the credentials to use for authentication.
             - If not specified, ANSIBLE_AZURE_AUTH_SOURCE environment variable will be used and default to C(auto) if variable is not defined.
-            - C(auto) will follow the default precedence of module parameters -> environment variables -> default profile in credential file
-              C(~/.azure/credentials).
-            - When set to C(cli), the credentials will be sources from the default Azure CLI profile.
-            - Can also be set via the C(ANSIBLE_AZURE_AUTH_SOURCE) environment variable.
+            - C(auto) will follow the default precedence of: module parameters -> C(env) -> C(credential_file) -> C(cli).
```

Comment:
No semicolon in the description statement。 The default value is defined in the module, you need to specify here and define the default value below.

Reply:
Please let me restate your request to ensure I understand it.

* I should change this line to remove everything after the colon so that it reads `C(auto) will follow the module default precedence`
* I should update the `notes` section below to define the default precedence order

Is this correct?